lib/chef/knife/cookbook_show.rb

```diff
@@ -51,6 +51,11 @@ class CookbookShow < Knife
         :description => "Show corresponding URIs"
 
       def run
+        cookbook_name, segment, filename = @name_args[0], @name_args[2], @name_args[3]
+        cookbook_version = @name_args[1] == "latest" ? "_latest" : @name_args[1]
```

Comment:
Will not hold this up for it, and it's totally up to your judgement, but I find avoiding the ternary operator where possible to make things more readable:

```
cookbook_version = @name_args[1]
cookbook_version = "_latest" if cookbook_version == "latest"
```